pkgs/development/python-modules/nbconflux/default.nix

```diff
@@ -17,7 +25,7 @@ buildPythonPackage rec {
   checkInputs = [ pytest responses ];
 
   checkPhase = ''
-    pytest tests
+    JUPYTER_PATH=${nbconvert}/share/jupyter pytest tests -k 'not (test_post_to_confluence or test_optional_components)'
```

Comment:
Please use pytestCheckHook.

Reply:
I can't get this env variable set during pytestCheckHook, and it fails without the env variable.